<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BBWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VlcWidgetVideo" name="video" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1061</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCar">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Car</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carLeft">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carForward">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
    <property name="shortcut">
     <string comment="w"/>
    </property>
   </widget>
   <widget class="QPushButton" name="carBack">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carRight">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCamera">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>260</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraUp">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>280</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraLeft">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraDown">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraRight">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QFrame" name="statusFrame">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>480</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ipLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Car IP:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_Local_File"/>
    <addaction name="actionOpen_URL"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionCar_Address"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Local_File">
   <property name="text">
    <string>Open Local File</string>
   </property>
  </action>
  <action name="actionOpen_URL">
   <property name="text">
    <string>Open URL</string>
   </property>
  </action>
  <action name="actionCar_Address">
   <property name="text">
    <string>Car Address</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header>vlc-qt/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
